classes/class.ilObjMumieTaskGUI.php

```diff
@@ -592,6 +596,58 @@ public function submitAvailabilitySettings()
         $this->performCommand($cmd);
     }
 
+    /**
+     *
+     */
+    public function displayUserList()
+    {
+        global $ilTabs;
+        $ilTabs->activateTab('userList');
+        $html = $this->initUserList();
+        $this->tpl->setContent($html);
+    }
+
+    private function initUserList()
+    {
+        global $ilCtrl, $lng;
+        require_once('./Customizing/global/plugins/Services/Repository/RepositoryObject/MumieTask/classes/forms/class.ilMumieTaskGradeOverviewFormGUI.php');
+        $form =  new ilMumieTaskGradeOverviewFormGUI();
+        $form->setTitle($lng->txt('rep_robj_xmum_frm_user_overview_list_search_title'));
+        $form->addCommandButton('displayUserList', $lng->txt('rep_robj_xmum_frm_user_overview_list_search'));
+        $form->setFormAction($ilCtrl->getFormAction($this));
+        $this->form = $form;
+        if (!$this->form->checkInput()) {
+            $this->tpl->setContent($this->form->getHTML());
+            return;
+        }
+        $this->tpl->addCss("./Customizing/global/plugins/Services/Repository/RepositoryObject/MumieTask/templates/mumie.css");
+        $this->form->setFields($this, $this->form);
+        $html = str_replace("ilTableOuter", "mumie-user-table", $this->form->getHTML());
+        return $html;
+    }
+
+    public function displayGradeList()
+    {
+        global $ilTabs, $ilCtrl, $lng;
+        $ilTabs->activateTab('userList');
+        if (!is_null($_GET['newGrade'])&& !is_null($_GET["user_id"])) {
+            $this->plugin->includeClass('class.ilMumieTaskGradeListGUI.php');
+            ilMumieTaskLPStatus::overrideGrade($this);
+            $cmd = 'displayUserList';
+            $this->performCommand($cmd);
+            return;
+        }
+        require_once('./Customizing/global/plugins/Services/Repository/RepositoryObject/MumieTask/classes/forms/class.ilMumieTaskGradeListFormGUI.php');
+        $form =  new ilMumieTaskGradeListFormGUI($this);
+        $form->setFields($this, $this->form);
+        $form->setFormAction($ilCtrl->getFormAction($this));
+        $form->addCommandButton('displayUserList', $lng->txt('rep_robj_xmum_frm_grade_overview_list_back'));
+        $this->form = $form;
+        $html = str_replace("ilTableOuter", "mumie-user-table", $this->form->getHTML());
```

Comment:
This should be the responsibility of `ilMumieTaskGradeListFormGUI`

Just override the parent classes getHtml method

e.g.

```

public function getHTML()
    {
        $html = parent::getHTML();
        return str_replace("ilTableOuter", "mumie-user-table", $html);
    }
```

Reply:
done